test/test_sparse.py

```diff
@@ -4723,15 +4782,10 @@ def test_gradcheck_mm(self, layout, dtype, device, masked, fast_mode):
                     r" grad: Strided, mat1: Sparse(Csc|Bsr|Bsc), mat2: Strided"
                     r"|addmm: computation on (CPU|CUDA) is not implemented for "
                     r"Strided \+ Sparse(Csc|Bsr|Bsc) @ Strided without MKL)"):
-                torch.autograd.gradcheck(mm, (x, y), check_sparse_nnz=True, fast_mode=fast_mode, masked=masked)
+                torch.autograd.gradcheck(mm, (x, y), fast_mode=fast_mode, masked=masked)
             self.skipTest('NOT IMPL')
         else:
-            if masked:
-                r = torch.autograd.gradcheck(mm, (x, y), check_sparse_nnz=True, fast_mode=fast_mode, masked=masked)
-            else:
-                # Specifying check_sparse_nnz is unnecessary in
-                # non-masked/sparse semantics
-                r = torch.autograd.gradcheck(mm, (x, y), fast_mode=fast_mode, masked=masked)
+            r = torch.autograd.gradcheck(mm, (x, y), fast_mode=fast_mode, masked=masked)
```

Comment:
nit: you don't need to assign/assert its value

Reply:
Fixed.